### PR TITLE
Divided Swift and ObjC library, added deployment  targets.

### PR DESCRIPTION
**Pull request by K-Be**
Added deployment targets for any supported platform. 
Added subspec for the ObjC library and for the Swift library.
Use Swift  subspec as a default spec.

### CODE REVIEW

File: KeepLayout.podspec [hunk 1]
```diff
@@ -9,6 +9,35 @@ Pod::Spec.new do |s|
...
+  s.default_subspec = 'Swift'
+
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
+  s.tvos.deployment_target = '9.0'
+
+  s.subspec 'ObjC' do |op|
+    op.source_files = 'Sources'
+    op.exclude_files = 'Sources/*.{swift}'
+    op.ios.deployment_target = '5.0'
```
Comment (Tricertops) [inline KeepLayout.podspec]: This is probably redundant? It is overriden by line `17`.

File: KeepLayout.podspec [hunk 2]
```diff
@@ -9,6 +9,35 @@ Pod::Spec.new do |s|
...
+    op.osx.deployment_target = '10.7'
+    op.tvos.deployment_target = '9.0'
+    op.ios.framework = 'UIKit'
+    op.tvos.framework = 'UIKit'
+    op.osx.framework = 'Cocoa'
+    op.osx.exclude_files = 'Sources/UIViewController+KeepLayout.{h,m}', 'Sources/UIScrollView+KeepLayout.{h,m}'
+  end
+  s.subspec 'Swift' do |sp|
+    sp.dependency 'KeepLayout/ObjC'
+    sp.source_files = 'Sources/*.{swift}'
```
Comment (Tricertops) [inline KeepLayout.podspec]: Swift variant of the library uses also `.m` files. Does it work correctly?

File: KeepLayout.podspec (hunk 1 above)
Comment (K-Be) [inline KeepLayout.podspec]: Yes, I need line 17 to build Swift spec, but I can build ObjC spec with lower deployment target.

File: KeepLayout.podspec (hunk 2 above)
Comment (K-Be) [inline KeepLayout.podspec]: Yes, they added with 33 line. 
I have next struct in the Pods project: 
Pods
    KeepLayout
          ObjC
          SupportFiles
          Swift

Also I've create a test project with one swift and one ObjC target (I've posted pod file from it), and all works file

**Comment by K-Be** [comment]
Also I can share my test project but don't think that it too complex to show it.

**Pull request merged**